website/src/main/resources/microsite/data/rules.yml

```diff
@@ -0,0 +1,13 @@
+rules:
+- ExplicitResultTypes
+- RemoveUnusedImports
+- RemoveXmlLiterals
+- ProcedureSyntax
+- DottyVolatileLazyVal
+- ExplicitUnit
+- DottyVarArgPattern
+- NoAutoTupling
+- NoValInForComprehension
+- Sbt1
+- NoInfer
+- Disable
```

Comment:
it's worth noting that we miss the latest `RemoveUnusedTerms` rule. We can add docs for it into a separate PR

Reply:
Good catch, I think we will eventually be able to automatically render this from the cli either as a resource generator or tut passthrough.